src/pet.js

```diff
@@ -0,0 +1,125 @@
+//magic numbers
+const MAX_FITNESS = 10;
+const MIN_HUNGER = 0;
+const HUNGRY = "I am hungry"
+const UNFIT = "I need a walk"
+const BOTH = "I am hungry AND I need a walk"
+const FINE = "I feel great!"
```

Comment:
These variables are only used in your checkUp function and should therefore be defined in it. You should try to keep your variables as local to where they are used as possible, to make your code easier to read and to keep your namespace clear